tools/packaging/Makefile

```diff
@@ -18,20 +18,10 @@ export SNAPCRAFT_FILE
 export VERSION_FILE
 export VERSIONS_YAML_FILE
 
-test:
-	@$(MK_DIR)/.ci/test.sh
```

Comment:
Is this no longer used?

Reply:
Not sure if it ever was.  test.sh doesn't exist in this repository.

Reply:
Ok, this must have been used while running the CI for 1.x packaging repo.